README.md

```diff
@@ -39,6 +39,31 @@ $ yarn dev
 
 **NB:** React components and CSS will hot load after changes (this can be confirmed by watching the JavaScript console), but changes to device drivers and other code outside of the React components will not - use 'Reload' to reload after such changes. If the compilation/hot reload of a component fails for any reason (e.g. from a syntax error) you may need to reinitialize the hot loader by reloading the application.
 
+### Docker for Linux
+
+If you are running Linux you probably need to be using an Ubuntu distribution or derivative. To get around this for other distrubutions you can try to build a local docker image which is based on Ubuntu 18.04 and use the yarn/npm commands interactively.
+
+**NOTE:** You need to add udev rules to your host for uploads to actually work. You can find the udev rules in this [gist](https://gist.github.com/gniezen/2a40b7c191a27de94962a49c46a602df). The file should be placed in `/etc/udev/rules.d/` and the host should be rebooted.
```

Comment:
I'm wondering if we shouldn't maybe put the udev rules in the uploader repo itself instead of a gist that may disappear in future. Maybe `resources/linux`?

Reply:
Agreed and done.